pkg/network/usm/monitor_test.go

```diff
@@ -719,25 +730,20 @@ func (s *USMHTTP2Suite) TestSimpleHTTP2() {
 			},
 		},
 		{
-			name: " / path with repeated string",
+			name: "path with repeated string",
 			runClients: func(t *testing.T, clientsCount int) {
 				clients := getClientsArray(t, clientsCount)
-				for i := 0; i < 1000; i++ {
+
+				// currently we have a bug with paths which are not Huffman encoded, therefor I am skipping them by string length 3.
```

Comment:
```suggestion
				// currently we have a bug with paths which are not Huffman encoded, therefore we are skipping them by string length 3.
```
Typo fix + avoid using first person tense in comments